src/spacel/aws/elb.py

```diff
@@ -30,3 +33,11 @@ def _elb_in_service(self, elb):
         except ClientError:
             pass
         return False
+
+    def _register_instance_elb(self, elb):
+        try:
+            self._elb.register_instances_with_load_balancer(
+                LoadBalancerName=elb['name'],
+                Instances=[{'InstanceId': self._instance_id}])
+        except ClientError:
+            pass
```

Comment:
The above refactor would remove the expected `instance is already registered` exception here, making this ClientError more likely to be `logger.warn`-worthy?